playbook/app/pb_kits/playbook/playbook-doc.js

```diff
@@ -104,7 +104,7 @@ import * as UserBadge from 'pb_user_badge/docs'
 import * as Walkthrough from 'pb_walkthrough/docs'
 import * as WeekdayStacked from 'pb_weekday_stacked/docs'
 
-WebpackerReact.setup({
+WebpackerReact.registerComponents({
```

Comment:
Question on this one.

Looking at the source code of the `.setup` function.... it appears it already calls the `registerComponents` function itself.


https://github.com/renchap/webpacker-react/blob/174cad7a129083a1d18f20c56774d3b35b402cbc/javascript/webpacker_react-npm-module/src/index.js#L64


What is the benefit of changing this?

Reply:
Officially, we should have never had to call `.setup` on this class. Unfortunately, until recent React library upgrades, this was necessary considering the `webpacker-react` repo has been dead for a while (approx 5 years). 

Now, when we call `WebpackerReact.registerComponents`, the components are rendered as expected and the lighthouse